widget/label_test.go

```diff
@@ -21,15 +21,15 @@ func TestLabel_MinSize(t *testing.T) {
 }
 
 func TestLabel_Text(t *testing.T) {
-	label := &Label{Text: "Test"}
```

Comment:
As above. We support both the `New<widget>()` and `&<widget>{}` formats, these test changes imply that the latter may no longer work?

Reply:
Andy, supporting both NewX() and &X{} is great, the problem is with "ExtendBaseWidget". If you support &X{} then basically all "func (x *X)" need to call "x.ExtendBaseWidget(x)" which is a code smell.

I'll revert these back to &X{} for this PR.

Reply:
Yes it is a tradeoff, but that is the decision we have made. The ExtendBaseWidget call is only required on methods that could be called before render - and that reference render features.

Thanks for reverting the changes. If there is a discussion to be had about removing support for this approach we should do so in a new ticket.